### ISSUE TITLE
Downloader urllib3.util.Retry `allowed_methods` arg causes TypeError: __init__() got an unexpected keyword argument 'allowed_methods'

### ISSUE THREAD

**Issue by TheAustinator** [saezlab/omnipath#18]
Code to reproduce:
```
import decoupler as dc
df = dc.get_dorothea(organism='human', levels=['A', 'B', 'C', 'D'])
```
Traceback:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
/tmp/ipykernel_425/1965394605.py in <module>
----> 1 df = dc.get_dorothea(organism='human', levels=['A', 'B', 'C', 'D'])

/usr/local/lib/python3.8/dist-packages/decoupler/omnip.py in get_dorothea(organism, levels, weight_dict)
    135 
    136     # Load Dorothea
--> 137     do = op.interactions.Dorothea.get(
    138         fields=['dorothea_level', 'extra_attrs'],
    139         dorothea_levels=['A', 'B', 'C', 'D'],

/usr/local/lib/python3.8/dist-packages/omnipath/_core/requests/_utils.py in wrapper(wrapped, _instance, args, kwargs)
    102     @wrapt.decorator(adapter=wrapt.adapter_factory(argspec_factory))
    103     def wrapper(wrapped, _instance, args, kwargs):
--> 104         return wrapped(*args, **kwargs)
    105 
    106     if hasattr(clazz, "get") and not hasattr(clazz.get, "__wrapped__"):

/usr/local/lib/python3.8/dist-packages/omnipath/_core/requests/_utils.py in _get_helper(cls, **kwargs)
     29         The result which depends the type of the request and the supplied parameters.
     30     """
---> 31     return cls()._get(**kwargs)
     32 
     33 

/usr/local/lib/python3.8/dist-packages/omnipath/_core/requests/interactions/_interactions.py in __init__(self)
    195 
    196     def __init__(self):
--> 197         super().__init__(InteractionDataset.DOROTHEA)
    198 
    199     @classmethod

/usr/local/lib/python3.8/dist-packages/omnipath/_core/requests/interactions/_interactions.py in __init__(self, datasets, exclude)
     66         exclude: Optional[Datasets_t] = None,
     67     ):
---> 68         super().__init__()
     69 
     70         datasets = _to_dataset_set(datasets, "datasets", set(InteractionDataset))

/usr/local/lib/python3.8/dist-packages/omnipath/_core/requests/_request.py in __init__(self)
     74 
     75     def __init__(self):
---> 76         self._downloader = Downloader(options)
     77 
     78     @classmethod

/usr/local/lib/python3.8/dist-packages/omnipath/_core/downloader/_downloader.py in __init__(self, opts)
     47         if self._options.num_retries > 0:
     48             adapter = HTTPAdapter(
---> 49                 max_retries=Retry(
     50                     total=self._options.num_retries,
     51                     redirect=5,

TypeError: __init__() got an unexpected keyword argument 'allowed_methods'
```
Versions:
```
omnipath==1.0.5
decoupler==1.1.0
urllib3==1.25.8
```
Code to fix:
Delete line 52 of `omnipath/_core/downloader/_downloader.py`, `allowed_methods=["HEAD", "GET", "OPTIONS"],`
Happy to make the PR, just lmk

**Comment by michalk8** [comment]
There's a `urllib3>=1.26.0` requirement, which should also be included in version `1.0.5`.
If you prefer merging #19 , think it should be fine as well.

**Comment by michalk8** [comment]
closed via #19
